gef-ui/styles/styles.scss

```diff
@@ -1 +1,19 @@
 @import "node_modules/govuk-frontend/govuk/all";
+@import "./helpers.scss";
+
+.govuk-section-break {
+  &--bold {
+    border: 0;
+    border-bottom: 3px solid $govuk-text-colour;
+  }
+}
+
+@media (min-width: 40.0625em) {
```

Comment:
What does this min-width refer too? I wasn't aware there were any responsive designs?

Reply:
So the prototype kit already has break points for mobile devices. I just stole one of the break points (40.0625em) and used it here.

Reply:
If the GEF design doesn't require breakpoints then best remove this. We're not designing for mobile or tablets afaik

Reply:
Even though we are not designing for mobile or tablet, Gov prototype is responsive, so when using all their patterns, the  application will naturally fit on a mobile device. By removing this breakpoint, this will be the only page which will break on smaller devices. If you happy for the page to go from this:
![Screenshot 2021-03-04 at 09 49 18](https://user-images.githubusercontent.com/14890432/109945597-4f926200-7ccf-11eb-94bc-6bc16553ea10.png)

to this:
![Screenshot 2021-03-04 at 09 50 36](https://user-images.githubusercontent.com/14890432/109945613-5620d980-7ccf-11eb-958f-283ead4376f1.png)

then i shall remove

Reply:
Just to let you know, I have removed the breakpoint. We can always re introduce it at a later stage

Reply:
I'd prefer any unnecessary code to be removed for clarity. Best confirm with Misha or Gavin that we're not concerned with mobile/tablet layouts.